.github/workflows/linux-eic-shell.yml

```
@@ -334,6 +334,25 @@ jobs:
           *.dot
           *.svg
         if-no-files-found: error
+    - uses: dawidd6/action-download-artifact@v2
+      with:
+        branch: ${{ github.event.pull_request.base.ref || github.ref_name }}
+        path: ref/
+        name: rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root
+        if_no_artifact_found: warn
+    - uses: actions/checkout@v3
+      with:
+        repository: 'eic/epic-capybara'
```

```suggestion
        repository: 'eic/epic-capybara'
        path: epic-capybara
```
to avoid https://github.com/eic/EICrecon/actions/runs/5606457235/jobs/10256831201?pr=773#step:10:26